### ISSUE TITLE
 Change checkbox to a radio button in Cleanup for converting to biblatex/bibatex

### ISSUE THREAD

**Issue by Siedlerchr** [JabRef/jabref#10583]
**Is your suggestion for improvement related to a problem? Please describe.**
Currently, two checkboxes are displayed for "Convert to BibTeX" and "Convert to biblatex" but one can only choose one. 

**Describe the solution you'd like**
The checkboxes should be replaced by radio buttons with a Toggle Group

**Additional context**

![grafik](https://github.com/JabRef/jabref/assets/320228/1831e58b-68d9-4072-b2cc-38d381630148)

**Comment by JawadTUE** [comment]
Can I be assigned to this issue?

**Comment by suxiaocha123** [comment]
Sorry , May I be assigned to this issue ?

**Comment by ThiloteE** [comment]
@JawadTUE @suxiaocha123 Are you from the same university? Are you working together as a team? If not, would it be permissible for you both to work together and co-author the PR?

In general we assign issues on first come first serve basis. It is already too late, but in future, please take care to not work on issues that are reserved (you can see it in the top right corner at the projects section), as grading requirements may differ and group work may not be permitted by some lecturers and in some courses.

**Comment by suxiaocha123** [comment]
I am sorry that I did not know the distribution mechanism before, and I am deeply sorry for the conflict caused. I don't think we went to the same college, but I was honored to work with J

**Comment by JawadTUE** [comment]
No, I don't know @suxiaocha123 . This seems like an issue that can be resolved by one person, so I'll look for another issue for myself.

**Comment by calixtus** [comment]
This is a non-issue.
See https://github.com/JabRef/jabref/pull/10601#issuecomment-1791542973

**Comment by koppor** [comment]
Longer explanation at https://github.com/JabRef/jabref/pull/10601#issuecomment-1791531950

TLDR: When radio buttons were used, we would need a third radio button (for no-op). Which makes the dialog even larger.
